featuretools/primitives/utils.py

```diff
@@ -134,6 +202,94 @@ def _get_descriptions(primitives):
     return descriptions
 
 
+def _get_summary_primitives(primitives: List) -> Dict[str, int]:
+    """Provides metrics for a list of primitives."""
+    controllable_list = {}
+    unique_input_types = set()
+    unique_output_types = set()
+    ct_multi_in = 0
+    ct_multi_out = 0
+    ct_extra_data = 0
+    ct_controllable = 0
+    # ct_trainable = 0  # get clarification
+    prim_input_ww_checks = {  # semantic tag for time index and logical types in inputs
```

Comment:
Does the list below include the full set of ww logical types? 

If so, this will need to be updated any time a new logical type is added. We should either add a test that will notify us when we need to update this list, or generate the logical types that make up this list by looking through `LogicalType.__subclasses__` or something that allows us to get the full set. One thing to look out for, though, is how we would want to handle user-generated logical types, since in theory a user could create their own logical type and then a primitive that uses it, and I'm not sure we want that logical type showing up here.